FWCore/Framework/src/EventProcessor.cc

```diff
@@ -1734,7 +1734,7 @@ namespace edm {
         }));
     WaitingTaskHolder afterProcessTask;
     if (subProcesses_.empty()) {
-      afterProcessTask = std::move(finalizeEventTask);
```

Comment:
This was intended to release the task and avoid its lifetime being too long

Reply:
`WaitingTaskHolder` doesn't seem to have an `operator=(WaitingTaskHolder&&)` ?

https://github.com/cms-sw/cmssw/blob/4bbb63b0f11ccf880dc738a7338165db26e42503/FWCore/Concurrency/interface/WaitingTaskHolder.h#L32-L49